src/components/Person/Person.jsx

```diff
@@ -1 +1,15 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+    {
+      person.age && <p className="Person__age">{`I am ${person.age}`}</p>
+    }
+    <p className="Person__partner">
+      {person.isMarried
+        ? `${person.partnerName} is my ${
+          person.sex === 'm' ? 'wife' : 'husband'
+        }`
+        : 'I am not married'}
```

Comment:
for improve readibilty of code make to separate ternary exprassions in component and just pass variable here
```
const partner =  sex === 'm' ? 'wife' : 'husband';
const personPartner = isMarried 
  ? `${partnerName} is my ${partner}`;
   : 'I am not married'
```
```suggestion
      {personPartner}
```